hosts/ghaf-log/configuration.nix

```diff
@@ -15,6 +16,7 @@
       (modulesPath + "/profiles/qemu-guest.nix")
       inputs.sops-nix.nixosModules.sops
       inputs.disko.nixosModules.disko
+      ./loki.nix
     ]
     ++ (with self.nixosModules; [
       common
```

Comment:
`service-nginx` module needs to be imported for default nginx configuration.

Reply:
Done.